lib/Models/Feature.js

```diff
@@ -92,7 +92,7 @@ Feature.fromEntityCollectionOrEntity = function(entity) {
     if (entity.entityCollection) {
         feature = entity.entityCollection.getById(entity.id);
     }
-    if (!feature || !feature instanceof Feature) {
+    if (!feature || !(feature instanceof Feature)) {
```

Comment:
Yes, very nice catch. It would be good to check the feature info panel for time-varying features (eg. in #test and #csv), and with some of the #polling test cases too.